app/assets/javascripts/qsets.js

```diff
@@ -94,3 +94,11 @@ function initQuestionFilter() {
     $('.no-questions > p').html(noQuestionsNotification);
   }
 }
+
+function showNoSubsetNotification() {
+  if ($('.qset').length == 0) {
+    $('.no-qsets').removeClass("hidden");
+    var noSubsetsNotification = 'There are no subsets. ' + '<a href="#" data-toggle="modal" data-target="#modal-new-qset">Create one now!</a>';
```

Comment:
I agree that it would be nice. @rckim77 @katloboda can you raise that as a separate trello item for a future fix (use the UI improvement label) and put it in the Doing Next column?